sycl/include/CL/sycl/half_type.hpp

```diff
@@ -400,6 +400,37 @@ class half {
     half r = *this;
     return -r;
   }
+
+// Operator +, -, *, /
+#define OP(op, op1)                                                            \
```

Comment:
Using op_eq instead of op1 would make it a bit more more readable:
```suggestion
#define OP(op, op_eq)                                                          \
```

Reply:
I have swapped op1 to op_eq